### PR TITLE
roles: biocache-service, nameindex update

### PR DESCRIPTION
**Pull request by JeGoi**
Change biocache-service config files installation from files to template
Remove hard coded ALA from config files

### CODE REVIEW

**Comment by JeGoi** [comment]
Add a fix for name matching

File: ansible/roles/biocache-service/tasks/main.yml [hunk 1]
```diff
@@ -68,36 +68,6 @@
         use_cache: true
         force_cache: true
         proxy_pass: "http://127.0.0.1:8080{{biocache_service_proxy_target}}/occurrence/legend"
-      - path: "{{biocache_service_context_path}}/occurrence/facets"
```
Comment (ansell) [inline ansible/roles/biocache-service/tasks/main.yml]: These removals need to be replaced with a configurable approach. The ALA rely heavily on these rules to cater for the amount of traffic that we serve on our low budget hardware.

File: ansible/roles/nameindex/tasks/main.yml [hunk 2]
```diff
@@ -48,21 +48,20 @@
   tags:
     - "nameindex"
 
-## Ensure that data_dir/lucene exists as a directory
-
+## Ensure that data_dir/lucene exists as a directory (ALA and not-ALA)
```
Comment (ansell) [inline ansible/roles/nameindex/tasks/main.yml]: Can you split the nameindex changes out into a separate pull request please?

File: ansible/roles/biocache-service/tasks/main.yml (hunk 1 above)
Comment (JeGoi) [reply ansible/roles/biocache-service/tasks/main.yml]: Right, I've miss them. They are not part of my modification.
I'll be back with an update.

File: ansible/roles/nameindex/tasks/main.yml (hunk 2 above)
Comment (JeGoi) [reply ansible/roles/nameindex/tasks/main.yml]: Right, I'll do it

Comment (djtfmartin) [reply ansible/roles/nameindex/tasks/main.yml]: thanks @JeGoi

**Comment by JeGoi** [comment]
I've created two new pull request for each and with the missing lines.
Sorry, for that unfortunate deletion and happy that @ansell have seen that.
